src/app/app.component.ts

```diff
@@ -1,12 +1,20 @@
 import { Component, OnInit, OnDestroy } from '@angular/core';
 
+export interface User {
+  name?: string;
+  age?: number;
+}
+
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
 export class AppComponent implements OnInit, OnDestroy {
   title = 'ngx-skeleton-loader-demo';
+  loading = true;
+  temp: User[] = [
```

Comment:
Can we use some better naming here, please?

Reply:
Remove the code.. those are some leftover codes that were no longer required.